react-ui/src/components/ListItemLink.jsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * Displaying row with external data in vertically stacked list
```

Comment:
Same comment as above. Also this leads to an internal page, not an external one.

Reply:
Edited